scripts/release-on-comment.py

```diff
@@ -119,19 +119,23 @@ def trigger_release(payload_path: Path, token: str) -> None:
 
         release_branch = f"release-{version}"
 
-        check_call(["git", "config", "user.name", "pytest bot"])
-        check_call(["git", "config", "user.email", "pytestbot@gmail.com"])
+        Command.run(["git", "config", "user.name", "pytest bot"])
+        Command.run(["git", "config", "user.email", "pytestbot@gmail.com"])
 
-        check_call(["git", "checkout", "-b", release_branch, f"origin/{base_branch}"])
+        Command.run(["git", "checkout", "-b", release_branch, f"origin/{base_branch}"])
 
         print(f"Branch {Fore.CYAN}{release_branch}{Fore.RESET} created.")
 
-        check_call(
+        Command.run(
             [sys.executable, "scripts/release.py", version, "--skip-check-links"]
         )
 
         oauth_url = f"https://{token}:x-oauth-basic@github.com/{SLUG}.git"
-        check_call(["git", "push", oauth_url, f"HEAD:{release_branch}", "--force"])
+        Command.run(["git", "push", oauth_url, f"HEAD:{release_branch}", "--force"])
+        if Command.has_error():
```

Comment:
Instead of checking if `completed_process.returncode != 0` after every run, we just run every command with `Command.run()`. And we just check once if any error occurred. Then we can raise an error from the first command that failed.